### DISCUSSION TITLE
purpose of the "user.md" ...

### DISCUSSION THREAD

**Discussion by nic0711** [danielmiessler/Fabric#207] [Q&A] [answered]
I haven't really understood the purpose of the "user.md" yet... many patterns have the file, some do not.

_My question/idea:_
(How) Is it possible to use this file to give the pattern additional, individual parameters?
e.g. by querying them?

Example:
```
Please fill in the following project-specific information to personalize the elaboration:
- [Project name]: _____________________
- List of involved companies/authorities/institutions]: _____________________
- Specific systems and processes in the current situation]: _____________________
- Specific project objectives]: _____________________
- Relevant legal frameworks (e.g., GDPR, HIPAA)]: _____________________
- Technology preferences (if any)]: _____________________

```
By answering these questions, it will be ensured that the output is tailored to the specific needs and requirements of the project.

**Comment by zestysoft** [comment, marked as answer]
https://support.netdocuments.com/s/article/System-Message-versus-User-Message
